salt/states/virt.py

```diff
@@ -775,9 +1018,9 @@ def pool_running(name,
                  username=None,
                  password=None):
     '''
-    Defines and starts a new pool with specified arguments.
+    Defines a new pool with specified arguments.
 
-    .. versionadded:: 2019.2.0
+    .. versionadded:: neon
```

Comment:
this will now need to be updated to sodium, as we already cut the neon rc release

Reply:
ACK